evidence/wirednkod/0001-2024-05-evidence.md

```diff
@@ -0,0 +1,42 @@
+# Evidence-0001: Retain at Rank 1
+
+|                 |                                                                         |
+| --------------- | ----------------------------------------------------------------------- |
+| **Report Date** | 11 May 2024                                                             |
+| **Submitted by**| Nikos Kontakis                                                          |
+
+## Member details
+
+- Matrix username: @wirednkod:matrix.org
+- Polkadot address: 15DCZocYEM2ThYCAj22QE4QENRvUNVrDtoLBVbCm5x4EQncr
+- Current rank: 1
+- Date of initial induction: Seeding
```

Comment:
Nikos was not part of the Fellowship [seeding](https://github.com/polkadot-fellows/seeding).
So, the Date of initial induction should read: "26 December 2023", as per Fellowship induction and promotion [proposal 50](https://collectives.polkassembly.io/referenda/50).